NIPTool/API/external/api/api_v1/endpoints/batches.py

```diff
@@ -169,7 +200,11 @@ def coverage(request: Request, batch_id: str, adapter: NiptAdapter = Depends(get
 
 @router.get("/batches/{batch_id}/report/{coverage}")
 def report(
-    request: Request, batch_id: str, coverage: str, adapter: NiptAdapter = Depends(get_nipt_adapter)
+    request: Request,
+    batch_id: str,
+    coverage: str,
```

Comment:
Can not see that this parameter is used

Reply:
It is being passed to the template on line 240.